<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndInventoryControl</class>
 <widget class="QMainWindow" name="wndInventoryControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventory Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Master Inventory List</string>
    </property>
    <widget class="QLabel" name="lblFilter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter by Category:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbCategoryFilter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>431</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrintLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Print Label</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveItem">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrintAll">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Print All</string>
     </property>
    </widget>
    <widget class="QTableView" name="tblInventory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>531</width>
       <height>191</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>19</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Add to Master Inventory List</string>
    </property>
    <widget class="QLabel" name="lblNewName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNewCategory">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNewDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNewName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNewCategory">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtNewDescription">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>461</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>122</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmbCategoryFilter</tabstop>
  <tabstop>tblInventory</tabstop>
  <tabstop>btnPrintLabel</tabstop>
  <tabstop>btnRemoveItem</tabstop>
  <tabstop>btnPrintAll</tabstop>
  <tabstop>txtNewName</tabstop>
  <tabstop>txtNewCategory</tabstop>
  <tabstop>txtNewDescription</tabstop>
  <tabstop>btnAdd</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>wndinventorycontrol</receiver>
   <slot>createItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrintAll</sender>
   <signal>clicked()</signal>
   <receiver>wndinventorycontrol</receiver>
   <slot>printLabelAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPrintLabel</sender>
   <signal>clicked()</signal>
   <receiver>wndinventorycontrol</receiver>
   <slot>printLabelCurrent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tblInventory</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>wndinventorycontrol</receiver>
   <slot>deleteItem(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbCategoryFilter</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>wndinventorycontrol</receiver>
   <slot>refreshMasterList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
